db/migrate/20240613151304_create_declarations.rb

```diff
@@ -0,0 +1,22 @@
+class CreateDeclarations < ActiveRecord::Migration[7.1]
+  def change
+    create_enum :declaration_states, %w[submitted eligible payable paid voided ineligible awaiting_clawback clawed_back]
+    create_enum :declaration_types, %w[started retained-1 retained-2 completed]
```

Comment:
are extended declarations only applicable to ECF?

Reply:
yes